packages/tracking/src/class-tracking.php

```diff
@@ -103,11 +101,9 @@ public function tracks_record_event( $user, $event_name, $properties = array(),
 			return false;
 		}
 		$terms_of_service = new Terms_Of_Service();
-		// Don't track users who have opted out or not agreed to our TOS, or are not running an active Jetpack.
-		if (
-			! $terms_of_service->has_agreed()
-			|| ! $this->connection->is_user_connected()
-		) {
+		$status           = new Status();
```

Comment:
Since we're now using `Status()`, should we import it at the top of the file?

Reply:
We don't need to do that because `Status` is in the same namespace as `Tracking` - all classes are assumed to be `\Automattic\Jetpack`.